### DISCUSSION TITLE
modpoll

### DISCUSSION THREAD

**Discussion by tomekl1978** [StealthChesnut/HA-FoxESS-Modbus#10] [Q&A]
Modpoll I have:

Protocol configuration: MODBUS/TCP, FC4
Slave configuration...: address = 247, start reference = 11000 (PDU), count = 16
Communication.........: 192.168.86.17, port 502, t/o 1.00 s, poll rate 1000 ms
Data type.............: 16-bit register, input register table

-- Polling slave...
Reply time-out!

What does this mean?

**Comment by Daibutt** [comment]
Hi @tomekl1978 
Is this with a RS485 >Lan converter?
The IP should be the IP of your RS485 converter not your Fox inverter, not sure if that is the issue?

**Comment by tomekl1978** [comment]
yes it is lan converter. The ip is RS485 converter.

**Reply by tomekl1978** [reply]
I can access it trough the website

**Reply by williamjeccles** [reply]
@tomekl1978 which converter are you using?

**Comment by tomekl1978** [comment]
usr w610. yesterday i tried to get to usr with modscan64 and it says timed out.

**Reply by Parisn0** [reply]
Realise this was a while ago, but if not yet resolved, my similar problems were caused by a faulty connection between RS485 converted and the inverter, check your wires and pin numbers
